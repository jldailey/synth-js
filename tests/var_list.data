{ users: [ { email: "a@b.com" } ] }
