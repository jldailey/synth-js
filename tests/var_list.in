span 'Email: @{users.0.email}'
