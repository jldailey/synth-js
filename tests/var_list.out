<span>Email: a@b.com</span>
